script.js

```diff
@@ -2,15 +2,264 @@ const mobileMenu = document.querySelector('#mobile-menu-modal');
 const hamburgerButton = document.getElementById('hamburger-button');
 const closeModal = document.querySelector('#modal-close-icon img');
 const mobileMenuLinks = document.querySelectorAll('.mobile-menu-link');
+const cardWorks = document.getElementById('card_works');
+const cardModel1Informations = [
+  'Multi-Post Stories',
+  "A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  ['css', 'html', 'bootstrap', 'Ruby'],
+  'See Project',
+];
+const cardModel2Informations = [
+  'Profesional Art Printing Data',
+  "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+  ['html', 'bootstrap', 'Ruby'],
+  'See Project',
+];
+const cardItem2Title = 'Data DashboardHealthcare';
+const cardItem3Title = 'Website Protfolio ';
+
+const projectsData = {
+  workModel1: {
+    title: cardModel1Informations[0],
+    description: cardModel1Informations[1],
+    techsList: cardModel1Informations[2],
+    project: cardModel1Informations[3],
+  },
+  workModel2: {
+    title: cardModel2Informations[0],
+    description: cardModel2Informations[1],
+    techsList: cardModel2Informations[2],
+    project: cardModel2Informations[3],
+  },
+  item2: {
+    title: cardItem2Title,
+  },
+  item3: {
+    title: cardItem3Title,
+  },
+  cardModel: {
+    classes: [
+      'work_model_1',
+      'work_model_2 item-1',
+      'work_model_2 item-2',
+      'work_model_2 item-3',
+      'work_model_2',
+      'work_model_2 item-2',
+      'work_model_2 item-3',
+    ],
+  },
+};
```

Comment:
Your projects information should be organized in an array containing objects, one object for each project. It should look like something like this: 

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```
The array should hold a number of objects which is the same number of your projects. 
This will make your code cleaner.